bot.py

```diff
@@ -209,6 +215,7 @@ def check(m):
 			await channel.send("That's incorrect, please guess again")
 		message = await bot.wait_for("message", check=check)
 		guess = message.content.split()[-1]
+		profile = await userprofile.load_profile(message.author.id)
```

Comment:
check if the guess is correct, _then_ load the profile. If it exists, break out from the loop, if it doesn't tell the user to run `p!begin`

Reply:
why should people without profiles be allowed to play the game at all? if the winner of a minigame doesn't have a profile and doesn't run `p!begin` won't that just ruin the game for everyone and stop the game from finishing?

Reply:
You can't prevent them from typing messages. I'm saying if someone guesses correctly, and they don't have a profile, they don't win.

Reply:
ah i see what you mean. i'll correct that